disk/assets/configuration/spec.yaml

```diff
@@ -54,6 +54,15 @@ files:
               type: array
               items:
                 type: string
+          - name: file_system_global_blacklist
```

Comment:
While I see where this is going, would these options be better as `init_config` options than `instance` variables?  A user seeing a `file_system_blacklist` and a `file_system_global_blacklist` may be confused?

Reply:
Good idea, but I think not because there are currently no such options there due to there only ever being a single instance (local machine).

Reply:
Well, conceivably a user could create multiple check instances ...  Putting global option to the init provides a logical distinction that they would apply everywhere.

Seeing these two options next to one another is confusing, which one should a user use?

    ## @param file_system_blacklist - list of strings - optional
    ## Instruct the check to not collect from matching file systems.
    ##
   

    ## @param file_system_global_blacklist - list of strings - optional
    ## Instruct the check to always add these patterns to `file_system_blacklist`.

Reply:
changed